### ISSUE TITLE
[BUG] Blank Canvas, Missing Report Template Error

### ISSUE THREAD

**Issue by Gary-Isaac** [aiverify-foundation/aiverify#315]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I built the latest Docker image and running it on AWS ec2. I get a no templates response (error 500) when trying to create a new project. I cannot figure out where the logs would be for the image. I would think that the 500 error will be logged by the web server. I am inexperienced with Docker images, apart from building and deploying, so I do not know where to look for the error logs.

If you can point me to the logs then I can give your more information about the error.

### Current Behavior

Create report page loads but templates do not.

### Expected Behavior

The page is supposed to load with a list of templates or use a blank canvas.

### Environment

```markdown
Used the Docker file. docker-compose.yml has:

version: '3.8'
image: aiverify:0.10.3


- Operating System and Version:
- AI Verify Version:
- Any other relevant information (e.g. version of related package dependencies installed):

Did you build using source code or from docker file? If from source code, please provide the below information:
- Python Version:
- Node Version:
- Redis Version:
- Mongo Version: 
- Screen Resolution:
```

### Screenshots/ Code snippets

<img width="926" alt="ai verify reports" src="https://github.com/user-attachments/assets/497de15b-a485-4a1e-9f38-3f8462c4a7cb">

**Comment by Gary-Isaac** [comment]
I understand the issue now and see the fix. I have to build from source and modify some configuration.

 Fix/scr 009 #232

**Comment by imda-benedictlee** [comment]
Hi @Gary-Isaac, glad to hear the issue has been resolved for you. I will close it. If you require additional assistance, feel free to open this issue.
